keras/losses.py

```diff
@@ -419,6 +419,61 @@ def __init__(
         super().__init__(mean_absolute_error, name=name, reduction=reduction)
 
 
+@keras_export("keras.losses.ScaleInvariantError")
+class ScaleInvariantError(LossFunctionWrapper):
+    """Computes the Scale Invariant Error between labels and predictions.
+
+    Formulas:
+
+    ```
+    g = log(y_pred) - log(y_true)
+    loss = alpha * sqrt(mean(g ** 2) - lamda * mean(g) ** 2)
+    ```
+
+    Standalone usage:
+
+    >>> y_true = [[1., 2.], [3., 4.]]
+    >>> y_pred = [[2., 4.], [6., 8.]]
+    >>> scale_inv_error = tf.keras.losses.ScaleInvariantError()
+    >>> loss = scale_inv_error(y_true, y_pred)
+    >>> loss.numpy()
+    2.6845474
+    """
+
+    def __init__(
+        self,
+        reduction=losses_utils.ReductionV2.AUTO,
+        alpha=10.0,
+        lamda=0.85,
+        name="scale_invariant_error",
+    ):
+        """Initializes `ScaleInvariantError` instance.
+
+        Args:
```

Comment:
Please fix indent (4 spaces)

Also: please move the Args section to the class docstring and remove this docstring.